### PR TITLE
[LLM] add convert's python deps for LLM

### PR DESCRIPTION
**Pull request by HensonMa**
Add quantize's required python deps for LLM (numpy, torch, transformers, and sentencepiece)

### CODE REVIEW

**Comment by rnwang04** [comment]
quantize don't need such deps but convert needs.
Not sure convert has any limitations for transformers? @plusbang

**Comment by HensonMa** [comment]
> quantize don't need such deps but convert needs. Not sure convert has any limitations for transformers? @plusbang

Oh, yes, it's this file's python deps. https://github.com/intel-analytics/BigDL/blob/main/python/llm/src/bigdl/llm/utils/convert_util.py

let me change the name of the deps group

File: python/llm/setup.py [hunk 1]
```diff
@@ -128,6 +129,10 @@ def setup_package():
 
     for url in lib_urls[platform_name]:
         download_libs(url, change_permission=change_permission)
+    
+    install_requires=[]
+    if "--all" in sys.argv:
+        install_requires += CONVERT_DEP
```
Comment (TheaperDeng) [inline python/llm/setup.py:135]: let's use `extra_requires` in metadata for this

Comment (HensonMa) [reply python/llm/setup.py]: resolved

**Comment by plusbang** [comment]
> quantize don't need such deps but convert needs. Not sure convert has any limitations for transformers? @plusbang

yeah, dependency `transformers` is necessary during converting.

**Pull request merged**